backend/routes.js

```diff
@@ -36,16 +73,19 @@ module.exports = [
     handler: async (request, h) => {
       const { email, password } = request.payload
       const user = usersDatabase.find(u => u.email === email)
-      
-      if (password === 'user123') {
-        const jwt = await createJsonWebToken(user)
-        return h
-          .response()
-          .state('my-jwt', jwt, cookieConfig)
-          .code(201)
-      }
 
-      return h.response().code(401)
+      return checkPassword(user.name, password).then(async (match) => {
```

Comment:
we should probably use async/await here as per the rest of the codebase + Hapi's standards